<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapEditor</class>
 <widget class="QWidget" name="MapEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>432</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Map Editor</string>
  </property>
  <layout class="QHBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>8</number>
   </property>
   <property name="margin" stdset="0">
    <number>9</number>
   </property>
   <item>
    <widget class="QGraphicsView" name="mapView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>300</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="sidePanelLayout">
     <property name="spacing">
      <number>8</number>
     </property>
     <property name="margin" stdset="0">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="mapListGroup">
       <property name="title">
        <string>Maps</string>
       </property>
       <layout class="QVBoxLayout" name="mapListLayout">
        <property name="margin" stdset="0">
         <number>8</number>
        </property>
        <item>
         <widget class="QListWidget" name="mapList"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="deleteMapButoon">
            <property name="text">
             <string>Delete Map</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addMapButton">
            <property name="text">
             <string>Add New Map</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="textureGroup">
       <property name="title">
        <string>Textures</string>
       </property>
       <layout class="QVBoxLayout" name="textureListLayout">
        <property name="margin" stdset="0">
         <number>8</number>
        </property>
        <item>
         <widget class="QListWidget" name="textureList"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="addTextureButton">
            <property name="text">
             <string>Add Texture</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteTextureButton">
            <property name="text">
             <string>Delete Texture</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Save All</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
